File: Venus/src/Presentation/Web/Cms/Venus.Presentation.Web.Cms/venus.presentation.web.cms.client/src/dtos/blogs/ReadBlogDto.ts
import { DynamicPropertiesDtoBase } from "../base/DtoBase";
import type { ReadUrlDto } from "../urls/ReadUrlDto";

export class ReadBlogDto extends DynamicPropertiesDtoBase{
    title:string;
    description:string;
    urlId:string;
    url:ReadUrlDto;
}

File: Venus/src/Core/Venus.Core.Application/Dtos/Entities/Blogs/ReadBlogDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Venus.Core.Application.Dtos.Base;
using Venus.Core.Application.Dtos.Systems.Urls;

namespace Venus.Core.Application.Dtos.Entities.Blogs
{
    public class ReadBlogDto : ReadVenusDtoBase
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public string JsonData { get; set; }
        public string UrlPath { get; set; }
        public ReadVenusUrlDto Url { get; set; }
    }

    public class ReadBlogDynamicData
    {
        public string Category { get; set; }
    }
}
